lib/test-requirements.txt

```diff
@@ -22,6 +23,7 @@ hypothesis>=6.17.4
 parameterized
 pytest
 pytest-cov
+requests>=2.27, <3
```

Comment:
Needed to re-iterate in `test-requirements.txt` otherwise requests resolves in min-requirements python test to 2.18